_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,267 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
+
+- **(not alone)** it is not the only [content][] in the [HTML document][]; and
+- **(changed)** the `innerText` property of the [element][html element] changes; and
+- **(no child changed)** the [element][html element] does not have [children][child] whose `innerText` property also changes; and
+- the [document][html document] the [element][html element] belongs to:
+  - **(ready)** has [readiness][document readiness] equal to "complete" before the change happens; and
+  - **(inactive)** is associated with a [window][] that has the [last activation timestamp][] equal to "positive infinity".
+
+## Expectation 1
+
+For the test target there exists a [mechanism][] that might be used to:
+
+- pause, stop or hide the change of the [visible text content][]; or
+- alter the frequency of the changes of the [visible text content][].
+
+**Note**: If there is more than one test target, a single [mechanism][] may be used to pause, stop, hide or alter the frequency for all test targets.
+
+## Expectation 2
+
+The existing [mechanism][] is [visible][], has an [accessible name][] that is not only [whitespace][], and is [included in the accessibility tree][].
+
+## Assumptions
```

Comment:
This rule needs an assumption that the changing text provides information.

Reply:
I'm not sure I agree with this one. 
One can argue that text is always information. But, even without arguing such, from the SC: "The intent of this Success Criterion is to avoid distracting users during their interaction with a Web page."
Even if the updating text does not provide information, it will (potentially) distract users. Thus, I would say that even that needs to be checked.

Reply:
Text can definitely be decorative (think of random words that float somewhere in the background). While I agree that such floating words could distract the user and that the intent document mentions that users should not get distracted, I don't think distracting users is actually a WCAG violation. The SC clearly states that only **information** that gets updated needs to have such a mechanism.

However I don't like this interpretation either so I am willing to drop it even though I think it is technicality correct.